components/admin-panel/sections/accounting/AccountingCategoriesTable.tsx

```diff
@@ -1,178 +1,77 @@
-  const allExpenseTypes = Object.values(omit(ExpenseType, 'FUNDING_REQUEST'));
-  const getOption = value => {
-    if (value === null) {
-      return { value, label: intl.formatMessage({ id: 'AllExpenses', defaultMessage: 'All expenses' }) };
-    } else {
-      return { value, label: i18nExpenseType(intl, value) };
-    }
-  };
-
-  // Build options list
-  const options = React.useMemo(() => [null, ...allExpenseTypes].map(getOption), [value]);
-
-  // View for accountants
-  if (table.options.meta.disabled) {
-    return (
-      <p>
-        {value === null
-          ? intl.formatMessage({ id: 'AllExpenses', defaultMessage: 'All expenses' })
-          : value.map(value => i18nExpenseType(intl, value)).join(', ')}
-      </p>
-    );
-  }
-
-  return (
-    <StyledSelect
-      isMulti
-      minWidth={200}
-      inputId={`${cell.id}-input`}
-      options={options}
-      isClearable={false}
-      value={Array.isArray(value) ? value.map(getOption) : null}
-      placeholder={intl.formatMessage({ id: 'AllExpenses', defaultMessage: 'All expenses' })}
-      onChange={options => {
-        if (!options?.length || options.find(option => option.value === null)) {
-          setValue(null);
-        } else {
-          setValue(uniq(options.map(option => option.value).sort()));
-        }
-      }}
-    />
-  );
+  onDelete: (category: AccountingCategory) => void;
 };
 
 const columns = [
   {
     accessorKey: 'code',
     header: () => <FormattedMessage id="AccountingCategory.code" defaultMessage="Code" />,
-    cell: EditableCell,
     meta: { input: { required: true, maxLength: 255 } },
+    cell: ({ cell }) => {
+      return (
+        <div className="inline-block rounded-xl bg-slate-50 px-2 py-1 font-bold text-slate-800">{cell.getValue()}</div>
+      );
+    },
   },
   {
     accessorKey: 'name',
-    header: () => <FormattedMessage id="Fields.name" defaultMessage="Name" />,
-    cell: EditableCell,
+    header: () => <FormattedMessage defaultMessage="Name <i>· Friendly name</i>" values={{ i: I18nItalic }} />,
     meta: { input: { required: true, maxLength: 255 } },
+    cell: ({ cell, row }) => {
+      return (
+        <div className="inline-block rounded-xl bg-slate-50 px-2 py-1 font-bold text-slate-800">
+          {cell.getValue()}
+          {row.original.friendlyName && (
+            <span className="font-normal italic text-slate-700">&nbsp;·&nbsp;{row.original.friendlyName}</span>
+          )}
+        </div>
+      );
+    },
   },
   {
-    accessorKey: 'friendlyName',
-    header: () => <FormattedMessage id="AccountingCategory.friendlyName" defaultMessage="Friendly name" />,
-    cell: EditableCell,
-    meta: { placeholderPath: 'name', maxLength: 255 },
+    accessorKey: 'kind',
+    header: () => <FormattedMessage id="Transaction.Kind" defaultMessage="Kind" />,
+    cell: ({ cell }) => {
+      return <FormattedMessage {...AccountingCategoryKindI18n[cell.getValue()]} />;
```

Comment:
@hdiniz I'm getting a crash there when trying locally. I don't think `FormattedMessage` can be used with dynamic props like that, you have to use:

```suggestion
      return intl.formatMessage(AccountingCategoryKindI18n[cell.getValue()]);
```

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/7b51be01-72dc-4183-9502-d6552c0cb3d9)